source/css/_common/components/post/post-header.styl

```diff
@@ -110,6 +110,11 @@
   }
 }
 
+.post-meta-break {
+  flex-basis: 100%;
+  height: 0;
```

Comment:
How about

```suggestion
  display: none;
```

Reply:
`post-meta-break` is used to make a line break within the flex element. `display: none` will not work

Reply:
I thought it was a completely hidden element.